### PR TITLE
Use php8 stable

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | ---
| License       | MIT
| Doc PR        | ---

This is only part of the development pipeline and should be considered a bugfix, thats why I target `4.4` branch.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,11 +26,11 @@ matrix:
           env: deps=high
         - php: 7.4
           env: deps=low
-        - php: 8.0snapshot
+        - php: 8.0
           services: [memcached]
     fast_finish: true
     allow_failures:
```
Comment (nicolas-grekas) [inline .travis.yml]: can we remove this also?

Comment (OskarStark) [reply .travis.yml]: Done

**Review by xabbuh — APPROVED** [review]

**Review by nicolas-grekas — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @OskarStark.

**Comment by derrabus** [comment]
Awesome!

**Pull request merged**